subsys/bluetooth/host/classic/l2cap_br.c

```diff
@@ -2608,6 +2608,9 @@ void bt_l2cap_br_chan_del(struct bt_l2cap_chan *chan)
 	/* Reset internal members of common channel */
 	bt_l2cap_br_chan_set_state(chan, BT_L2CAP_DISCONNECTED);
 	BR_CHAN(chan)->psm = 0U;
+	if (L2CAP_BR_CID_IS_DYN(BR_CHAN(chan)->rx.cid)) {
+		BR_CHAN(chan)->rx.cid = 0U;
```

Comment:
need to reset all rx bt_l2cap_br_endpoint？

 will tx enpoint be the potential problems？